muni_portal/core/views/api/webpush.py

```diff
@@ -25,3 +27,15 @@ def create(self, request, *args, **kwargs):
         except Exception as e:
             logger.error(e)
             return Response(status=status.HTTP_400_BAD_REQUEST, data={"message": e})
+
+
+class VapidApiView(RetrieveAPIView):
+
+    permission_classes = [IsAuthenticated]
```

Comment:
no need to be authenticated - please relax this and simplify the test to skip authentication